common/tools/dev-tool/src/commands/admin/list/tasks/update-package.ts

```diff
@@ -0,0 +1,50 @@
+import * as path from "node:path";
```

Comment:
Should this be checked in? Seems very one-off'ish 

In fact, I almost wonder if the tasks directory should be `.gitignore`d or something - what do you think?

Reply:
Right. I wanted to have some example as the basis of future tasks. Adding to a gitignored folder wouldn't help with that.  Now I moved the content into comments.